SparkPost/2web2ui
# EO-737 Add friendly label to tables for Mailbox Providers

### What Changed
 - Added a function that converts the id of the mailbox provider to a more friendly label.
 - Moved the Mailbox Providers constants to the generic constants file.

### How To Test
- [Point your local upstreams to staging or production](https://confluence.int.messagesystems.com/display/ENG/Configuring+Openresty+Upstreams)
- Log into appteam or account 108 for data.
- Go to signals Health Score or Engagement Recency.
- Change facets to `Mailbox Providers`. Check that the cells in the table show the friendly name. 

### To Do
- [x] Review Feedback
- [ ] ~Maybe add friendly page header labels~
